roles/zooplus/tasks/globalprotect.yml

```diff
@@ -0,0 +1,23 @@
+---
+
+- name: Check if globalprotect client is installed
+  stat:
+    path: /usr/bin/globalprotect
+  register: globalprotect_existence
+
+- name: Copy globalprotect package
+  copy:
+    src: files/{{ globalprotect_package }}
+    dest: /home/{{ user }}/SW/{{ globalprotect_package }}
```

Comment:
I would use /tmp or ~/Downloads. It's a install binary

Reply:
/tmp
done